api/Worker.json

```
@@ -126,6 +135,59 @@
             "deprecated": false
           }
         },
+        "es2015_module": {
+          "__compat": {
+            "description": "Support for ECMAScript2015 module worker",
+            "support": {
+              "chrome": {
+                "version_added": "80"
+              },
+              "chrome_android": {
+                "version_added": "80"
+              },
+              "edge": {
+                "version_added": "80"
+              },
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "ie": {
+                "version_added": false
+              },
+              "nodejs": {
+                "version_added": "12.17.0",
+                "partial_implementation": true,
+                "notes": "ES2015 is enabled based on file extension rather than type option."
+              },
+              "opera": {
+                "version_added": null
```

Mapped based on the underlying engine version

```suggestion
                "version_added": "67"
```